src/pr-info.ts

```diff
@@ -527,14 +519,18 @@ function downloadsToPopularityLevel(monthlyDownloads: number): PopularityLevel {
 }
 
 export async function getOwnersOfPackage(packageName: string, oid: string, fetchFile: typeof defaultFetchFile): Promise<string[] | null | Error> {
-    const indexDts = `${oid}:types/${packageName}/index.d.ts`;
-    const indexDtsContent = await fetchFile(indexDts, 10240); // grab at most 10k
-    if (indexDtsContent === undefined) return null;
-    let parsed: HeaderParser.Header;
+    const packageJson = `${oid}:types/${packageName}/package.json`;
+    const packageJsonContent = await fetchFile(packageJson, 10240); // grab at most 10k
+    if (packageJsonContent === undefined) return null;
+    let packageJsonObj;
     try {
-        parsed = HeaderParser.parseHeaderOrFail(indexDtsContent);
+        packageJsonObj = JSON.parse(packageJsonContent);
     } catch (e) {
-        if (e instanceof Error) return new Error(`error parsing owners: ${e.message}`);
+        if (e instanceof Error) return new Error(`error parsing owners from package.json: ${e.message}`);
+    }
+    const parsed = HeaderParser.validatePackageJson(packageName, packageJsonObj, []);
```

Comment:
I assume the error here is because this uses the new infra?

Reply:
Which, metapoint, when does this repo get republished? Presumably we can change dt-tools and this doesn't get updated, which feels strange.

I sorta wonder how much of the mergebot could be moved to GHA...